.python-version

```diff
@@ -1 +1 @@
-3.8.12
+3.11.7
```

Comment:
Not that important but this is a minor version lower than the version in Dockerfile

Reply:
I copied this from h. Dependabot updates the Dockerfile but it doesn't know about `.python-version`, so we'll have this minor skew across our projects.

Reply:
I don't think we've ever seen problems due to Python minor version differences between what we use in dev and what the Docker container uses, but perhaps this could be bumped periodically by the cookiecutter or something? CC @seanh .